android/src/main/java/co/nimblehq/ic/kmm/suv/android/ui/components/NpsAnswer.kt

```diff
@@ -12,35 +12,50 @@ import androidx.compose.ui.Modifier
 import androidx.compose.ui.draw.alpha
 import androidx.compose.ui.graphics.Color
 import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.semantics.contentDescription
+import androidx.compose.ui.semantics.semantics
 import androidx.compose.ui.text.font.FontWeight
 import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.unit.dp
 import co.nimblehq.ic.kmm.suv.android.R
 import co.nimblehq.ic.kmm.suv.android.ui.theme.AppTheme
 import co.nimblehq.ic.kmm.suv.android.ui.theme.Typography
+import co.nimblehq.ic.kmm.suv.domain.model.Answer
+import co.nimblehq.ic.kmm.suv.domain.model.AnswerInput
+import co.nimblehq.ic.kmm.suv.domain.model.Answerable
 
 private const val NPS_SIZE = 10
 private val NPS_VIEW_HEIGHT = 56.dp
 private val NPS_VIEW_WIDTH = 345.5.dp
 
+object NpsAnswerContentDescription {
+    private const val NPS = "NPS"
+
+    fun nps(index: Int) = "$NPS-$index"
+}
+
+@Suppress("NAME_SHADOWING")
 @Composable
 fun NpsAnswer(
-    currentIndex: Int = -1,
-    onIndexChange: (Int) -> Unit,
+    answers: List<Answerable>,
+    onInputChange: (AnswerInput) -> Unit,
+    input: AnswerInput? = null
 ) {
-    var selectedIndex by remember { mutableStateOf(currentIndex) }
+    var input by remember { mutableStateOf(input) }
```

Comment:
same

Reply:
Fixed in https://github.com/suho/kmm-ic/pull/123/commits/07f6a3df329b5ed79084a3fe821669529cd47d33